src/modules/0.0.19/aws_account/entity/index.ts

```diff
@@ -17,7 +13,28 @@ export class AwsAccountEntity {
 
   @Column()
   secretAccessKey: string;
+}
+
+@Entity()
+export class AwsRegions {
+  @PrimaryGeneratedColumn()
+  id: number;
 
   @Column()
+  @cloudId
   region: string;
```

Comment:
I think the `id` might need to be just autogenerated number but not `primary` and let the region column be the PK if we want to join by the region name

Reply:
We can still add a secondary index on this to allow a join, can't we? I'm pretty sure you're allowed to have a foreign key relation onto a secondary index.

Reply:
I'll switch this, anyways.